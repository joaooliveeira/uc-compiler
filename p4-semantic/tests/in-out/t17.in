int v[] = {1, 2, 3, 4};

int main() {
    float x = 3.0;
    for (int i = 1; i <= 4; i++)
        v[x] = i;
    return 0;
}