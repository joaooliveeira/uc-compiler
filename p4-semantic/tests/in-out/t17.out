SemanticError: type(float) must be of type(int) @ 6:11
